akka-actor-typed-tests/src/test/scala/akka/actor/typed/receptionist/LocalReceptionistSpec.scala

```diff
@@ -44,29 +44,29 @@ class LocalReceptionistSpec extends TypedSpec with Eventually with StartSupport
   "A local receptionist" must {
 
     "must register a service" in {
-      val ctx = new EffectfulActorContext("register", behavior, 1000, system)
-      val a = Inbox[ServiceA]("a")
-      val r = Inbox[Registered[_]]("r")
+      val ctx = new BehaviorTestkit("register", behavior)
+      val a = TestInbox[ServiceA]("a")
+      val r = TestInbox[Registered[_]]("r")
       ctx.run(Register(ServiceKeyA, a.ref)(r.ref))
-      ctx.getEffect() // watching however that is implemented
+      ctx.requestEffect() // watching however that is implemented
       r.receiveMsg() should be(Registered(ServiceKeyA, a.ref))
-      val q = Inbox[Listing[ServiceA]]("q")
+      val q = TestInbox[Listing[ServiceA]]("q")
       ctx.run(Find(ServiceKeyA)(q.ref))
-      ctx.getAllEffects() should be(Nil)
+      ctx.requestAllEffects() should be(Nil)
```

Comment:
not sure that is a good name? is it because it's mutating that `get` is not a good name? hm

Reply:
I think i am used to the probe we use for streams, i didn't like get as it was destructive. WDYT?

Reply:
I switched this to retrieve based on the comments below